app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Display all the users</h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'style' %>
+
+<div class="container">
+  <% @users.each do |user| %>
+    <a href="<%= user_path(user.id) %>" class="userLink">
+      <img src="<%= asset_path('im12.jpg') %>" alt="User Photo">
+      <div class="userCon">
+        <h2><%= user.name %></h2>
+        <p>Number of posts: <%= user.posts.count %></p>
+      </div>
+    </a>
+  <% end %>
```

Comment:
- [ ] According to the project requirement, your design layout is meant to look exactly to the wireframe, but looking at your current design, that seems to be the case. I'll suggest you update this to follow the design guide. :smile: :wave: 

- - Your current design:
![Screenshot from 2023-11-24 08-45-48](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/2082537f-36f8-4f2f-ac08-4c20bebe0281)

- - Wireframe:
![Screenshot from 2023-11-24 08-45-37](https://github.com/faisalakb/Rails-Blog-App/assets/107632780/d51bc4ed-0514-44ad-bfae-062f7977de25)

- [ ] Also the users' photo `<img src="<%= asset_path('im12.jpg') %>" alt="User Photo">` is meant to come from the photo URL the user s=added while creating his/her account and not an image asset file. I'll suggest you update this too. :+1: :wave: